src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.cpp

```diff
@@ -567,18 +585,47 @@ KerrSchild::IntermediateVars<DataType, Frame>::get_var(
 }
 
 template <typename DataType, typename Frame>
-tnsr::ii<DataType, 3, Frame>
+Scalar<DataType> KerrSchild::IntermediateVars<DataType, Frame>::get_var(
+    const IntermediateComputer<DataType, Frame>& computer,
+    gr::Tags::TraceExtrinsicCurvature<DataType> /*meta*/) {
+  return trace(
+      get_var(computer, gr::Tags::ExtrinsicCurvature<3, Frame, DataType>{}),
+      get_var(computer, gr::Tags::InverseSpatialMetric<3, Frame, DataType>{}));
+}
+
+template <typename DataType, typename Frame>
+tnsr::I<DataType, 3, Frame>
 KerrSchild::IntermediateVars<DataType, Frame>::get_var(
     const IntermediateComputer<DataType, Frame>& computer,
-    gr::Tags::ExtrinsicCurvature<3, Frame, DataType> /*meta*/) {
-  return gr::extrinsic_curvature(
-      get_var(computer, gr::Tags::Lapse<DataType>{}),
-      get_var(computer, gr::Tags::Shift<3, Frame, DataType>{}),
-      get_var(computer, DerivShift<DataType, Frame>{}),
-      get_var(computer, gr::Tags::SpatialMetric<3, Frame, DataType>{}),
-      get_var(computer,
-              ::Tags::dt<gr::Tags::SpatialMetric<3, Frame, DataType>>{}),
-      get_var(computer, DerivSpatialMetric<DataType, Frame>{}));
+    gr::Tags::TraceSpatialChristoffelSecondKind<3, Frame, DataType> /*meta*/) {
+  const auto& inverse_spatial_metric =
+      get_var(computer, gr::Tags::InverseSpatialMetric<3, Frame, DataType>{});
+  const auto& d_spatial_metric =
+      get_var(computer, DerivSpatialMetric<DataType, Frame>{});
+  const size_t grid_size = get_size(get<0, 0, 0>(d_spatial_metric));
+  TempBuffer<
```

Comment:
What do you think of adding the two Christoffel symbols as separate `IntermediateVars`? That way when someone almost inevitably wants them in the future they're already implemented in a way that makes them easy to retrieve

Reply:
done